### DISCUSSION TITLE
Is mods metadata files created using MultiMc/PolyMc compatible with Prism Launcher?

### DISCUSSION THREAD

**Discussion by EchoEllet** [PrismLauncher/PrismLauncher#2103] [Q&A] [answered]
Is mods metadata files created using MultiMc/PolyMc compatible with Prism Launcher?

the files that has extension `toml` which exists in the `.index` (metadata)

How do I regenerate them using PrismLauncher if they didn't exists or created using another launcher which is a fork of MultiMc

**Comment by Aces-and-Jacks** [comment, marked as answer]
I'm pretty sure the files in .index is mainly used for updating. So if you update the mod and select from where then I'm pretty sure it'll regenerate the .index file. If Prism Launcher can't find the mod even tho it's there, your either on an incompatible version of the mod or you'll need to redownload it from the website you got it on through the launcher.

**Reply by TayouVR** [reply]
pretty much this, there is currently no way to *just* generate the metadata for your current mods on demand. All you can do is check for updates, which will then try to find the mods on either mod provider and fill in the metadata. I think if you then cancel the update task it would probably be as if you just filled in the metadata. So far there hasn't been a need to generate those metadata files without actually doing some online action like updating as well.
